### PR TITLE
sync growthepie project directory with oss-directory

### PR DESCRIPTION
**Pull request by mseidlx**
Hi OSO team, 
we compared our internal database against the latest version of the oss-directory and put the diff in this PR :)
Let me know if we should make any changes.

Thanks so much for your help!

### CODE REVIEW

**Comment by mseidlx** [comment]
We can also take out the new chain names that I added in case this creates overhead -- just thought they might come in handy in the future but definitely not necessary for us to have them in here for now.

File: data/projects/g/galxe.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-version: 5
```
Comment (ryscheng) [inline data/projects/g/galxe.yaml:1]: @mseidlx I think this is preventing the CI validating from passing. We use version numbers to handle schema migrations FYI

**Comment by ryscheng** [comment]
This is awesome!! Thanks so much for submitting this!! 
Could you help just make sure that `pnpm validate` passes? Happy to merge once `ci-default` passes

**Comment by mseidlx** [comment]
Good idea, will get that in tomorrow!

**Comment by mseidlx** [comment]
@ryscheng added websites to `project.json` schema and updated the project files accordingly

**Comment by ryscheng** [comment]
You're amazing!! Thank you so much!!

**Pull request merged**